examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -4153,7 +4152,7 @@ rtError pxScriptView::getContextID(int /*numArgs*/, const rtValue* /*args*/, rtV
 
 rtError pxScriptView::makeReady(int numArgs, const rtValue* args, rtValue* /*result*/, void* ctx)
 {
-  rtLogInfo(__FUNCTION__);
+  rtLogDebug(__FUNCTION__);
```

Comment:
Let's leave this log statement as is because it's rarely logged and it's helpful for debugging.